docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -96,6 +94,9 @@ Analysis takes minutes to hours to run depending on the size of the application
 +
 [TIP]
 ====
-{ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time.  With more cluster capacity, more analysis processes can be executed in parallel.
+{ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time. With more cluster capacity, more analysis processes can be executed in parallel.
 ====
-. When analysis is complete, click the *Report* link to see the results of the analysis.
\ No newline at end of file
+. When analysis is complete, click the *Report* link to see the results of the analysis.
+. Optional: Display the details of the analysis:
+.. Click the Options menu ({kebab}).
+.. Select *Analysis details*. The details are displayed in the *Analysis details for customers* window. You can choose either the YAML or JSON format.
```

Comment:
Why is it called "Analysis details for customers"? I think I've never seen it called like that before

Reply:
@abrugaro, no idea... This is what was in the original version of the doc. How should it be corrected? Or shall we just remove this sentence if it doesn't convey the truth?

Reply:
"Analysis details window"  is good enough

Reply:
Now I'm thinking that it's even better to remove this part since the first sentence aready says "click Analysis details" and it should be obvious that by clicking it, the details are displayed.

Reply:
Agreed